zipa/resource.py

```diff
@@ -99,7 +99,9 @@ def default_response_handler(response):
     def _prepare_entity(self, response):
         response_handler = self.config.response_handler or (lambda r: r)
 
-        parsed_response = response_handler(response) or {}
+        parsed_response = response_handler(response)
+        if parsed_response is None:
```

Comment:
Why don't we return `None`?
If somebody parse the response and return `None` I guess that this was their intend.